### PR TITLE
Add Gemfile and local dev instructions to README

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Added Gemfile to codify dependencies to run Jekyll site
- Added instructions to README on how to run the site

## QA

- Pull this branch
- Follow README instructions
- Verify you can run site locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,32 @@ A collection of documents that describe best practices for Canonical web team.
...
+## Run site locally
+
+This site is built using [Jekyll](https://jekyllrb.com/).
+
+To run this site locally, you'll need to first [install Ruby](https://www.ruby-lang.org/en/documentation/installation/) on your local machine.
+
+Once you have installed Ruby, you can then install [Bundler](https://bundler.io) by running;
+
+```
+gem install bundler
```
Comment (b-m-f) [inline README.md]: To get the correct version for the Gemfile.lock we can change this to `gem install bundler -v 1.17.3`

File: README.md [hunk 2]
```diff
@@ -4,6 +4,32 @@ A collection of documents that describe best practices for Canonical web team.
...
+The next step is to install the Ruby gems required to run the site with Bundler;
+
+```
+bundle install
+```
+
+You can then serve the site using Jekyll;
+
+```
+jekyll serve
```
Comment (b-m-f) [inline README.md]: I encountered a problem here, where `~/.gem/bin` was not in my path. Should we add a section for these things?

Comment (barrymcgee) [reply README.md]: Could you please add a 'Troubleshooting' section at the bottom of the README;

- Describe your problem including the error message you encountered
- Outline the steps to resolve it

Can you add this via another PR please?

File: README.md (hunk 1 above)
Comment (barrymcgee) [reply README.md]: Updated, thanks.

File: README.md (hunk 2 above)
Comment (b-m-f) [reply README.md]: Will do :)

**Pull request merged**